audio/mp3val/files/Makefile-cxxflags.patch

```diff
@@ -0,0 +1,8 @@
+--- Makefile.linux.orig	2007-03-12 17:50:16.000000000 +0100
++++ Makefile.linux	2021-09-07 18:04:00.000000000 +0200
+@@ -1,4 +1,4 @@
+-CXXFLAGS=-Wall -O2
++CXXFLAGS := -Wall -O2 $(CPPFLAGS)

```

Comment:
I don't think this is correct... why not just remove this line and just use what MacPorts sets (e.g., it should be using `-Os` and not `-O2`).